installing/installing_openstack/installing-openstack-installer-custom.adoc

```diff
@@ -18,6 +18,21 @@ processes.
 
 * Have metadata service enabled in {rh-openstack}
 
+[IMPORTANT]
+====
+{product-title} 4.3 is supported for use with {rh-openstack} 13. {rh-openstack}
+16 will also be supported when it is released; it is currently in the RC phase
```

Comment:
What does RC stand for?

Reply:
@ahardin-rh it stands for *release candidate*. I can go ahead and add that with the abbreviation, for those that don't know what that means. E.g., `release candidate (RC)`.